Simulator/postgres/database.js

```diff
@@ -1,50 +1,29 @@
-const {Client} = require('pg');
+const {Pool} = require('pg');
 
-const client = new Client();
+const client = new Pool();
+client.connect();
 
-module.exports.getHouses = async function(callback) {
+module.exports.getHouses = async function() {
```

Comment:
Should all database methods go under different exports?
Why not do:
```Javascript
Database = {
     method1: function(){},
     method2: function(){},
}

module.exports = Database
```

Reply:
Same same but different. But can change it so we get a more consistent layout.